data/features/m/m.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+m() {
```

Comment:
Headers missing. Is really necessary that I am telling you this? Did you not know already?

Reply:
solved